client/src/nv_ingest_client/nv_ingest_cli.py

```diff
@@ -67,6 +68,8 @@
 @click.option("--client_host", default="localhost", help="DNS name or URL for the endpoint.")
 @click.option("--client_port", default=6397, type=int, help="Port for the client endpoint.")
 @click.option("--client_kwargs", help="Additional arguments to pass to the client.", default="{}")
+@click.option("--client_type", default="rest", type=click.Choice(["rest", "simple"], case_sensitive=False),
+              help="Client type used to connect to the ingest service.")
```

Comment:
This is a bit confusing. The issue is that from the CLI's perspective, its using either a SimpleClient or a RESTClient. The CLI has no way of submitting Redis native requests to a Redis broker at the moment.

Historically, this was because we'd originally planning to standardize only on the REST interface and leave the choice of broker to the REST provider.

In contrast, the docker-compose configuration has Redis/Simple, because that dictates how the pipeline source is configured.